src/lib/components/primitives/input/input.svelte

```diff
@@ -42,7 +42,7 @@
     class={`${inputClasses} tw_input-base dark:tw_input-base_dark tw_focusable-element dark:tw_focusable-element_dark tw_flex-1 tw_w-full`}
     id={key}
     on:change={onChange}
-    placeholder={placeholder || label}
+    {placeholder}
```

Comment:
Removing the `label` fallback, and use the literal space character above, so that we don't end up with a duplication of label and placeholder.